include/opendht/http.h

```diff
@@ -108,7 +108,7 @@ class OPENDHT_PUBLIC Connection : public std::enable_shared_from_this<Connection
     std::string read_bytes(size_t bytes = 0);
     std::string read_until(const char delim);
 
-    void async_connect(std::vector<asio::ip::tcp::endpoint>&& endpoints, ConnectHandlerCb);
+    void async_connect(std::vector<asio::ip::tcp::endpoint>&& endpoints, ConnectHandlerCb, bool save_power = false);
```

Comment:
Add `set_keepalive()` method and `keepalive_` property to allow users to customize connection keepalive settings independently from async_connect.